src/coreclr/gc/unix/cgroup.cpp

```diff
@@ -68,18 +70,13 @@ class CGroup
 
         if (s_cgroup_version == 1)
         {
-            s_mem_stat_n_keys = 4;
-            s_mem_stat_key_names[0] = "total_inactive_anon ";
-            s_mem_stat_key_names[1] = "total_active_anon ";
-            s_mem_stat_key_names[2] = "total_dirty ";
-            s_mem_stat_key_names[3] = "total_unevictable ";
+            s_mem_stat_n_keys = 1;
```

Comment:
Because we're only reading a single field, we no longer need this array. Can you refactor the code?

Reply:
thank you, removed the code that is no longer used